src/util/string.h

```diff
@@ -85,9 +85,166 @@ std::string writeFlagString(u32 flags, const FlagDesc *flagdesc, u32 flagmask);
 size_t mystrlcpy(char *dst, const char *src, size_t size);
 char *mystrtok_r(char *s, const char *sep, char **lasts);
 u64 read_seed(const char *str);
+void parseTextString(const std::string &value, std::string &text, std::string &params, 
+		const char sep, const char esc='\\');
 bool parseColorString(const std::string &value, video::SColor &color, bool quiet,
 		unsigned char default_alpha = 0xff);
 
+struct ColorContainer {
```

Comment:
Same here. Struct declaration is OK in the header, but the definition should really be in `string.cpp`.